mmv1/third_party/terraform/services/gemini/resource_gemini_data_sharing_with_google_setting_binding_test.go.tmpl

```diff
@@ -53,50 +52,44 @@ func TestAccGeminiDataSharingWithGoogleSettingBinding_update(t *testing.T) {
 func testAccGeminiDataSharingWithGoogleSettingBinding_basic(context map[string]interface{}) string {
 	return acctest.Nprintf(`
 data "google_project" "project" {
-    provider = google-beta
 }
 
 resource "google_gemini_data_sharing_with_google_setting" "basic" {
-    provider = google-beta
     data_sharing_with_google_setting_id = "%{data_sharing_with_google_setting_id}"
     location = "global"
     labels = {"my_key" = "my_value"}
     enable_preview_data_sharing = true
 }
 
 resource "google_gemini_data_sharing_with_google_setting_binding" "basic_binding" {
-    provider = google-beta
     data_sharing_with_google_setting_id = google_gemini_data_sharing_with_google_setting.basic.data_sharing_with_google_setting_id
     setting_binding_id = "%{setting_binding_id}"
     location = "global"
     target = "projects/${data.google_project.project.number}"
+	product = "GEMINI_CLOUD_ASSIST"
```

Comment:
```suggestion
```
Let's remove this. Just want to make sure unset the field does not cause diff now. thanks!